src/scss/patternlib.scss

```diff
@@ -6,6 +6,9 @@
 @import '../styles/page-template/examples/block-areas/block-areas';
 @import '../patterns/question/examples/question-anatomy/pl-question-anatomy';
 
+// Find a support centre specific styles (experimental)
```

Comment:
Does it need to have its own style imported here instead of in the component itself?

Reply:
It is not a component. More a pattern. It is also experimental, so was unsure where to best place the styles associated.

Reply:
I've left a comment further up about the support centre specific styles. If they are needed then it would become a component without it's own page (index.njk) and then called from your patterns page. Essentially it would just be a folder with a `scss` file in it. 

The `patternlib.scss` file is only for styles specific to the design system website so wouldn't be bundled and deployed to the cdn. 

But as I said in a previous comment I'm not sure this requires it's own stylesheet.